tasks/eva_2722/prepare_covid19_processed_analysis_file.py

```diff
@@ -26,12 +26,19 @@ def prepare_processed_analyses_file(project, batch_size, processed_file_director
     total_analyses = total_analyses_in_project(project)
     logger.info(f"total analyses in project {project}: {total_analyses}")
 
-    processed_analyses = get_processed_analyses_files(processed_file_directory)
-    logger.info(f"no of processed analyses : {len(processed_analyses)}")
-    processed_files_with_no_analyses = processed_analyses.copy()
+    processed_analyses_files = get_processed_analyses_files(processed_file_directory)
```

Comment:
yes, depending upon which directories we are looking for files, it will be either run_ref or analysis_accession. 
I have updated the variable name and excluded analyses and also added field name as a parameter, so that we can explicitly specify which field we are looking up